src/components/InputV2/BigInt.tsx

```diff
@@ -0,0 +1,96 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { TextInput, TextInputProps } from './Text'
+
+const isBigInt = value => typeof value === 'bigint'
```

Comment:
does  `typeof <FilecoinNumber> === bigint`? If not, I think we should figure out how to support FilecoinNumber.

Reply:
That's a great idea, I think supporting FilecoinNumber as a prop makes a lot of sense!